python/core/auto_generated/qgscadutils.sip.in

```diff
@@ -87,10 +87,6 @@ Defines constraints for the :py:func:`QgsCadUtils.alignMapPoint()` method.
 
         QgsCadUtils::AlignMapPointConstraint xConstraint;
         QgsCadUtils::AlignMapPointConstraint yConstraint;
-
-        QgsCadUtils::AlignMapPointConstraint zConstraint;
```

Comment:
Not used today, but maybe it's worth to keep it. I don't know.

Reply:
Yes, I removed the commit.